doc/make-class-pages.py

```diff
@@ -5,7 +5,10 @@
 xenapi_json = "data/xenapi.json"
 classes_root = "content/xen-api/classes/"
 
-def template(c):
+releases_yml = "data/releases.yml"
+releases_root = "content/xen-api/releases/"
+
+def class_template(c):
```

Comment:
woof! the dog sniffed some possible issues in the file :)

Reply:
Well well... I'll rename it then :)